src/cli/credrank.js

```diff
@@ -73,4 +97,38 @@ function printCredSummaryTable(credGraph: CredGraph) {
   sortedParticipants.slice(0, 20).forEach((n) => console.log(row(n)));
 }
 
+function printCredDiffTable(credGraph: CredGraph, priorCredGraph: CredGraph) {
+  console.log(`# Top Participants By New Cred`);
+  const priorParticipants: Map<Uuid, Participant> = new Map();
+  for (const participant of priorCredGraph.participants())
+    priorParticipants.set(participant.id, participant);
+
+  const credParticipants = Array.from(credGraph.participants());
+  const sortedParticipants = sortBy(credParticipants, (p) => -p.cred);
+  function row({cred, description, id}) {
+    const prior = priorParticipants.get(id);
+    if (!prior)
+      throw `Participant [${description}, ${id}] exists in the new scores but not in the old. Rerun without -d to refresh.`;
+    const percentageChange = (100 * (cred - prior.cred)) / prior.cred;
+    const percentageChangeStr =
+      percentageChange > 10000
+        ? ">10,000%"
+        : `${percentageChange > 0 ? "+" : ""}${percentageChange.toFixed(1)}%`;
```

Comment:
I recommend using d3.format rather than hand rolling. (We already have this dep.)
https://github.com/d3/d3-format

Reply:
accepted